CMakeLists.txt

```diff
@@ -41,11 +41,20 @@ project(tritonpytorchbackend LANGUAGES C CXX)
 #   - Set TRITON_PYTORCH_DOCKER_IMAGE to use the docker image of
 #     PyTorch to base the build off.
 #
+# TRITON_PYTORCH_ENABLE_TORCHTRT: enables Torch-TRT runtime support.
+#   With TRITON_PYTORCH_DOCKER_IMAGE: set if to OFF for images before 21.11
+#   Without TRITON_PYTORCH_DOCKER_IMAGE:
+#     If you leave it ON: set TRITON_PYTORCH_TORCHTRT_LIB_PATH to Torch-TRT runtime lib. 
+#
+
 option(TRITON_ENABLE_GPU "Enable GPU support in backend" ON)
 option(TRITON_ENABLE_STATS "Include statistics collections in backend" ON)
+option(TRITON_PYTORCH_ENABLE_TORCHTRT "Enable TorchTRT support" ON)
```

Comment:
Add a check for TRITON_PYTORCH_ENABLE_TORCHTRT so that TRITON_PYTORCH_ENABLE_TORCHTRT  is not ON when TRITON_ENABLE_GPU is set to OFF

Reply:
Re Jetson build - currently, we do not build pytorch_backend at all, so no changes are needed at this time.

Reply:
But we should still have this check like we do not the ORT backend (as I pointed to before)

Reply:
No we should not  - now it's like 'off only' switch if user wants to turn it off. User never should specify it if he wants it ON.

Reply:
How does an off only switch work? If the user specifies TRITON_ENABLE_GPU=OFF we want to throw an error and not enable TRITON_PYTORCH_ENABLE_TORCHTRT just because the pytorch container includes the torch trt libs.

Reply:
Not enable - Yes. Error-no. Unless I can tell user set TRITON_PYTORCH_ENABLE_TORCHTRT to ON explicitly, not just used all defaults ? Unless we can distinguish them, I think the best course of action is to adjust  the flags silently.

Reply:
>  Unless I can tell user set TRITON_PYTORCH_ENABLE_TORCHTRT to ON explicitly, not just used all defaults ?

My perspective is the same that the user should control the option and we should not change these settings automatically. 
@deadeyegoodwin as well believes we should not adjust the flags silently.

Reply:
This is actually very good question - how exactly user set this flag when building the whole server?  Currently I am setting it in server's build.py but it's probably not the best idea - should be set in tritonserver repo.